SU2_CFD/include/solver_structure.hpp

```diff
@@ -12742,27 +12742,40 @@ class CDiscAdjFEASolver : public CSolver {
   su2double *E_i,               /*!< \brief Values of the Young's Modulus. */
             *Nu_i,              /*!< \brief Values of the Poisson's ratio. */
             *Rho_i,             /*!< \brief Values of the density (for inertial effects). */
-            *Rho_DL_i;          /*!< \brief Values of the density (for volume loading) */
-  su2double *Local_Sens_E, *Global_Sens_E,            /*!< \brief Local and global sensitivity of the Young's modulus. */
-            *Local_Sens_Nu, *Global_Sens_Nu,          /*!< \brief Local and global sensitivity of the Poisson ratio. */
-            *Local_Sens_Rho, *Global_Sens_Rho,        /*!< \brief Local and global sensitivity of the density. */
-            *Local_Sens_Rho_DL, *Global_Sens_Rho_DL;  /*!< \brief Local and global sensitivity of the volume load. */
-  su2double *Total_Sens_E,       /*!< \brief Total sensitivity of the Young's modulus (time domain). */
-            *Total_Sens_Nu,      /*!< \brief Local and global sensitivity of the Poisson ratio. */
-            *Total_Sens_Rho,     /*!< \brief Local and global sensitivity of the density. */
-            *Total_Sens_Rho_DL;  /*!< \brief Local and global sensitivity of the volume load. */
-
-  bool de_effects;              /*!< \brief Determines if DE effects are considered. */
-  unsigned short nEField;       /*!< \brief Number of electric field areas in the problem. */
-  su2double *EField;            /*!< \brief Array that stores the electric field as design variables. */
-  su2double *Local_Sens_EField, *Global_Sens_EField; /*!< \brief Local and global sensitivity of the Electric Field. */
-  su2double *Total_Sens_EField;
-
-  bool fea_dv;                  /*!< \brief Determines if the design variable we study is a FEA parameter. */
-  unsigned short nDV;           /*!< \brief Number of design variables in the problem. */
-  su2double *DV_Val;            /*!< \brief Value of the design variables. */
-  su2double *Local_Sens_DV, *Global_Sens_DV;          /*!< \brief Local and global sensitivity of the Design Variable. */
-  su2double *Total_Sens_DV;
+            *Rho_DL_i;          /*!< \brief Values of the density (for volume loading). */
+  int       *AD_Idx_E_i,        /*!< \brief Derivative index of the Young's Modulus. */
```

Comment:
It seems that for FSI adjoints the derivatives w.r.t. structural variables need extraction via local indexes (don't ask me why).